sdk/js/package-lock.json

```diff
@@ -19,7 +19,7 @@
         "axios": "^0.24.0",
         "bech32": "^2.0.0",
         "js-base64": "^3.6.1",
-        "near-api-js": "^0.45.1"
+        "near-api-js": "^0.44.2"
```

Comment:
Did you mean to go backwards?

Reply:
I bumped it down, because there was an incompatibility issue with the wallet-selector package (in the example UI), but it looks like that was fixed here, so I'll bump it back up. https://github.com/near/wallet-selector/pull/455